pkg/hub/controllers/slice_controller_unit_test.go

```diff
@@ -163,7 +161,7 @@ func TestReconcileToUpdateWorkerSlice(t *testing.T) {
 	}{
 		context.WithValue(context.Background(), types.NamespacedName{Namespace: "kube-slice", Name: "kube-slice"}, controllerSlice),
 		reconcile.Request{NamespacedName: types.NamespacedName{Name: "test-slice", Namespace: "kubeslice-system"}},
-		reconcile.Result{RequeueAfter: time.Second * 10},
+		reconcile.Result{RequeueAfter: time.Second * 120},
```

Comment:
can we change the reconcileInterval in tests to reduce test times. Like we did earlier ?

Reply:
This is a UT, not IT. There is no requeue and thus reconcileInterval is of no significance here. I just adjusted  the expected value to the default value.